ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift

```diff
@@ -173,6 +178,43 @@ class RootViewController: UIViewController, UITableViewDataSource, UITableViewDe
         return tableView
     }()
 
+    private(set) lazy var searchProgressSpinnerSwitchView: UIView = {
+        let itemRow = UIStackView()
+        itemRow.axis = .horizontal
+        itemRow.distribution = .equalCentering
+        itemRow.alignment = .leading
+        itemRow.isLayoutMarginsRelativeArrangement = true
+        itemRow.layoutMargins = UIEdgeInsets(top: 10, left: 10, bottom: 10, right: 10)
+        itemRow.translatesAutoresizingMaskIntoConstraints = false
+
+        let label = Label(style: .subhead, colorStyle: .regular)
+        label.text = "Show search spinner when user hits search"
+        itemRow.addArrangedSubview(label)
+
+        let searchSpinnerSwitch = UISwitch()
+        searchSpinnerSwitch.isOn = false
+        searchSpinnerSwitch.addTarget(self, action: #selector(shouldShowSearchSpinner(switchView:)), for: .valueChanged)
+
+        itemRow.addArrangedSubview(label)
+        itemRow.addArrangedSubview(searchSpinnerSwitch)
+
+        let itemsContainer = UIView()
+        itemsContainer.backgroundColor = Colors.tableBackground
+        itemsContainer.addSubview(itemRow)
+        itemsContainer.translatesAutoresizingMaskIntoConstraints = false
+
+        NSLayoutConstraint.activate([
+            itemsContainer.topAnchor.constraint(equalTo: itemRow.topAnchor),
+            itemsContainer.bottomAnchor.constraint(equalTo: itemRow.bottomAnchor),
+            itemsContainer.leadingAnchor.constraint(equalTo: itemRow.leadingAnchor),
+            itemsContainer.trailingAnchor.constraint(equalTo: itemRow.trailingAnchor)
```

Comment:
searchSpinnerSwitchLabel.centerYAnchor.constraint(equalTo: itemsContainer.centerYAnchor),
			searchSpinnerSwitch.centerYAnchor.constraint(equalTo: itemsContainer.centerYAnchor)

Justo to align the label and toggle switch...

Reply:
Added constraints and now they looks aligned.

---
In reply to: [470763039](https://github.com/microsoft/fluentui-apple/pull/174#discussion_r470763039) [](ancestors = 470763039)